rest-api/model/biobank_order.py

```diff
@@ -116,3 +157,52 @@ class BiobankOrderedSample(Base):
   collected = Column('collected', UTCDateTime)
   processed = Column('processed', UTCDateTime)
   finalized = Column('finalized', UTCDateTime)
+
+
+class BiobankOrderHistory(BiobankOrderBase, Base):
+  __tablename__ = 'biobank_history'
+
+  version = Column('version', Integer, primary_key=True)
+
+  @declared_attr
```

Comment:
You shouldn't need all of these here. They are already inherited from BiobankOrderBase.

Reply:
oops. Think I yanked and pasted instead of delete and pasted. Removed.